ui/packages/catalog-ui-search/src/main/webapp/react-component/location/location.less

```diff
@@ -65,3 +65,28 @@
     margin-top: 10px;
   }
 }
+
+@{reactNamespace}direction {
```

Comment:
I think before, this had the "child combinator selector"
` @{reactNamespace}direction > .toggle-input {`

and `direction.js` had the `toggle-input` class. I think the difference is that the former way ^^ is saying "apply this style to all `toggle-input`s but only if they're direct children of a `@{reactNamespace}direction`. now we're applying the style to all `@{reactNamespace}direction`s. Maybe in implementation these are one in the same - but maybe we can verify this